### DISCUSSION TITLE
[백준 1010] 다리 놓기

### DISCUSSION THREAD

**Discussion by BanApp** [ghdcksgml1/Algorithm_Study#123] [1일 1알고리즘]
<img width="946" alt="스크린샷 2022-03-09 오전 10 23 29" src="https://user-images.githubusercontent.com/93313445/157354049-a751f2bd-836d-429c-9f1f-4c1f7066d383.png">

```
pypy3

import sys
import math

n = int(sys.stdin.readline())

for _ in range(n):
    a, b = map(int, sys.stdin.readline().split())
    ans = math.factorial(b) // (math.factorial(a) * math.factorial(b - a))
    print(ans)
```

백준 알고리즘 분류에는 DP가 들어가 있지만 이걸 조합 라이브러리나 재귀로 풀면 시간초과가 날것 같았다.
그래서 그냥 조합공식으로 풀었다.

**Comment by ghdcksgml1** [comment]
해당 공식을 이용해서 DP로 풀어보세용 ㅋㅋ

<img width="235" alt="스크린샷 2021-11-14 오후 2 49 22" src="https://user-images.githubusercontent.com/79779676/141669331-5ea94a56-f70d-4063-a76c-90b67f8119d8.png">

확실히 실버 5 문제는 아님;;

**Reply by BanApp** [reply]
dp로만 풀리는 문제였으면 골드5 예상합니다ㅋㅋㅋ
